### PR TITLE
Fran new field for registrations

### PR DESCRIPTION
**Pull request by jfescobar18**

### CODE REVIEW

File: UnitTests/CMSWebTests/TestUtils/ContexTestUtils.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        public static IDictionary Items;
+        private const string Url = "https://localhost.tpsdb.com";
+
+        internal static CMSDataContext CurrentDatabase()
+        {
+            Items = new Dictionary<string, object>();
+            var c = FakeHttpContext();
+            HttpContextFactory.SetCurrentContext(c);
+            var dbname = $"CMS_" + Util.Host;
+            var dbExists = DbUtil.CheckDatabaseExists(dbname).Equals(DbUtil.CheckDatabaseResult.DatabaseExists);
```
Comment (brondavies) [inline UnitTests/CMSWebTests/TestUtils/ContexTestUtils.cs]: All of this database setup should be handled by the `DatabaseFixture` by decorating the test class with `[Collection("Database collection")]` See https://github.com/bvcms/bvcms/blob/develop/UnitTests/CMSWebTests/Code/QueryModelTests.cs#L9

File: UnitTests/CMSWebTests/Areas/OnlineReg/Models/NewAskChurchQuestionTest.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+namespace CMSWebTests.Areas.OnlineReg.Models.AskChurch
+{
+    public class NewAskChurchQuestionTest
+    {
+        public static bool BuildDb = true;
+        public static bool DropDb = false;
+        public static IDictionary Items;
+        private const string Url = "https://localhost.tpsdb.com";
+        private static readonly HttpClient client = new HttpClient();
```
Comment (brondavies) [inline UnitTests/CMSWebTests/Areas/OnlineReg/Models/NewAskChurchQuestionTest.cs]: None of these variables/constants look like they are used. Am I missing something?

**Comment by brondavies** [comment]
@jfescobar18 Will you address the conflicts and review comments on this PR?  Thanks.

**Comment by brondavies** [comment]
@jfescobar18 Just need you to resolve the conflicts here and we can merge

**Comment by jfescobar18** [comment]
> @jfescobar18 Just need you to resolve the conflicts here and we can merge

@brondavies I think you were talking about prev. comments

**Comment by brondavies** [comment]
> > @jfescobar18 Just need you to resolve the conflicts here and we can merge
> 
> @brondavies I thought you were talking about prev. comments

Yes, those too - but the tests are failing, will you look into that issue?

**Pull request merged**